uiowa/uiowa
# Add access to menu settings for content editors

Resolves #4703.
Resolves #4563. 
Resolves #4544. 
Resolves #4794. 

# To test
```
ddev restart # Only necessary if you have not done so since #4854 was merged.
composer install
blt frontend
blt ds --site sandbox.uiowa.edu
```
- Spot check LB style settings to make sure that they are rendering correctly in block forms and that selected values are reflected in block display.
- Add or edit a menu block on a new or existing page.
- When adding a menu block, "Vertical" should be the default option for menu orientation.
- Otherwise, the "Number of levels to display should only allow 1-3 as options.
- You should be required to select either "Vertical" or "Horizontal."
- ~After you selected the "Horizontal" option and saved the block, if you configure the block again and choose "Vertical" this time you see 1 as the selected value under "Number of levels to display."~
- If you set the style to "Horizontal" in any layout outside of the one column layout, the design should appear as a vertical menu.
- In Layout Builder the preview for the menu should always show the placeholder text.
- If the menu block is placed in a one column layout, the placeholder style should not change.

## To test the test
- `ddev phpunit docroot/modules/custom/layout_builder_custom --filter testMenuBlockOverrides`
- Confirm the test runs without errors or failures.
- Click the last link to see the HTML generated during the test.
- Visually confirm that the form matches the expected fields and order that you observed in previous testing.
